thirdparty/turbo/CMakeLists.txt

```diff
@@ -12,9 +12,9 @@ assert_var_defined(LDFLAGS)
 
 ep_get_source_dir(SOURCE_DIR)
 
-set(BUILD_CMD sh -c "${KO_MAKE_RECURSIVE} CC=\"${CC}\" CFLAGS=\"${CFLAGS}\" LDFLAGS=\"${LDFLAGS}\" all")
+set(BUILD_CMD "${KO_MAKE_RECURSIVE} CC=\"${CC}\" CFLAGS=\"${CFLAGS}\" LDFLAGS=\"${LDFLAGS}\" all")
 
-set(PATCH_CMD "${ISED} 's|CC=|CC:=|' ${SOURCE_DIR}/Makefile")
+set(PATCH_CMD sh -c "${ISED} 's|CC=|CC:=|' ${SOURCE_DIR}/Makefile")
```

Comment:
??

Reply:
It's used to build a shell one-liner below on Android ;).